README.md

```diff
@@ -285,6 +285,37 @@ Since a full Raspberry Pi 4 support is only available in upstream Linux kernels
 
 Once your Raspberry Pi 4 is happily running an EVE image you can start using EVE controller for further updates (so that you don't ever have to take an SD card out of your board). Build your rootfs by running `make ZARCH=arm64 HV=xen rootfs` (or `make ZARCH=arm64 HV=kvm rootfs` if you want KVM by default) and give resulting `dist/arm64/current/installer/rootfs.img` to the controller.
 
+### How to use on an Onlogic FR201 ARM board
+
+Onlogic Factor 201 (FR201) is a board based on the Raspberry Pi Compute Module 4 (CM4). To use EVE on FR201, build an image for Raspberry Pi 4, as described above and flash it on a USB stick. Then, to enable FR201's specific devices, the boot configuration has to be manually edited.
+
+Specifically, in file `config.txt` of the 1st partition of the live/installer image, the following 2 lines must be changed:
+
+```
```

Comment:
Maybe a diff would be easier to read (just a suggestion, feel free to accept or not): 

```diff
@@ -30,7 +30,7 @@ enable_uart=1
 # but for advantech uno-220 we need to set ce to 0x00
 # https://github.com/Advantech-IIoT/UNO-220-POE-/tree/master/srcs/dtbo/tpm#notes
 # PS: Comment this line for OnLogic FR201 device
-dtoverlay=bcm2711-spi-tpm-slb9670,ce=0x01
+#dtoverlay=bcm2711-spi-tpm-slb9670,ce=0x01
 
 # Disable warning overlays as they don't work well together with linux's graphical output
 avoid_warnings=1
@@ -49,4 +49,4 @@ include extraconfig.txt
 
 # Uncomment for the following line for OnLogic FR201
 # Please, don't forget to disable overlay bcm2711-spi-tpm-slb9670
-#include fr201.txt
+include fr201.txt
```